apps/cloud/src/api/blobs.rs

```diff
@@ -143,6 +155,31 @@ impl Context {
             ErrorStatus::InternalServerError.into_response()
         }
     }
+
+    #[instrument(skip(self))]
+    async fn get_user_resource_usage(&self, user_id: String) -> Result<Usage, ErrorStatus> {
+        info!("get_user_resource enter");
+        let Ok(workspace_id_list) = self.db.get_user_owner_workspaces(user_id).await else {
+            return Err(ErrorStatus::InternalServerError);
+        };
+        let mut total_size = 0;
+        for workspace_id in workspace_id_list {
+            let Ok(size) = self
+                    .storage
+                    .blobs()
+                    .get_blobs_size(workspace_id.to_string())
+                    .await else {
```

Comment:
I'd like to pass more error messages here, could that be done in the current implementation?  `let size = self.storage.blobs().get_blobs_size(workspace_id.to_string()).await.map_err(|err| ErrorStatus::InternalServerError(err))?;`

Reply:
Ok, it has been modified.